services/merak-network/grpc/service/service.go

```diff
@@ -99,6 +99,7 @@ func (s *Server) NetConfigHandler(ctx context.Context, in *pb.InternalNetConfigI
 			}
 		}()
 
+		wg.Wait()
```

Comment:
@yanmo96 Why you need to add `wg.Wait()` here?

Reply:
Originally it was designed to do services and register node then do network at the same time. But that was a miss understanding. Will fix how service work.  
Now add the wait there so it will take in the Alcor url first, then run the register node and network create.